c7n/actions.py

```diff
@@ -378,6 +373,57 @@ def process(self, resources, event=None):
             params['Payload'] = utils.dumps(payload)
             result = client.invoke(**params)
             result['Payload'] = result['Payload'].read()
+            if isinstance(result['Payload'], bytes):
+                result['Payload'] = result['Payload'].decode()
+            results.append(result)
+        return results
+
+
+class FunctionInvoke(EventAction):
```

Comment:
The reason the CI is failing on this function invoke action is that it's missing the `permissions` set, in this case since we're invoking an arbitrary python function, permissions should be an empty set.

Reply:
@thisisshi Thanks! I'm gonna re-submit